docs/content/docs/dev/datastream/fault-tolerance/serialization/custom_serialization.md

```diff
@@ -56,18 +56,6 @@ ListStateDescriptor<Tuple2<String, Integer>> descriptor =
 checkpointedState = getRuntimeContext().getListState(descriptor);
 ```
 {{< /tab >}}
-{{< tab "Scala" >}}
-```scala
-class CustomTypeSerializer extends TypeSerializer[(String, Integer)] {...}
-
-val descriptor = new ListStateDescriptor[(String, Integer)](
-    "state-name",
-    new CustomTypeSerializer)
-)
-
-checkpointedState = getRuntimeContext.getListState(descriptor)
-```
-{{< /tab >}}
```

Comment:
After remove scala part, we just have one tab. I suggest that we removing the top-level `tab`. Maybe replace it with code format(e.g. ```java)